acceptance-tests/mssql_test.go

```diff
@@ -105,6 +105,46 @@ var _ = Describe("MSSQL", Label("mssql"), func() {
 		golangAppTwo.DELETE(schema)
 	})
 
+	It("can be accessed by a JAVA app using the JDBC URL when require ssl is disabled", Label("JDBC-mssql"), func() {
```

Comment:
thought: this could be tested in the previous test by updating the instance and rebinding. It would probably be faster to update than to create the instance, although it is just a theory. The cons is that it is not as clear that we are testing this.

Reply:
Thank you for bringing this up, @pivotal-marcela-campo. Changing the `rds.forcessl` db parameter involves a restart in the instance. This is because in MSSQL the db parameter is established as `Apply type: static`, this behaviour is different from, for example, MySQL or Aurora, which means I couldn't update the service. I wanted to update the instance.

Do you think it is worth creating an entry in the documentation mentioning this case?

![image](https://github.com/cloudfoundry/csb-brokerpak-aws/assets/9041034/32140ebd-5deb-45b0-86fd-ca5dfb942feb)

Reply:
It might be useful information, although we don't document each parameter that forces a restart, so for consistency I wouldn't not for now.